pkg/node/node_address.go

```diff
@@ -404,3 +402,7 @@ func GetNodeAddressing(enableIPv4 bool) *models.NodeAddressing {
 		},
 	}
 }
+
+func init() {
+	InitDefaultPrefix("")
```

Comment:
Maybe leave a note here explaining the non-default/user-specified version of this is re-initialised in daemon/main.go::initEnv.

Reply:
fixed